src/api/ResponseBuilder.ts

```diff
@@ -0,0 +1,81 @@
+import Discord, { Client, Message, TextChannel } from "discord.js";
+
+const WARN_COLOR = "#e3bb39";
+const FEEDBAP_COLOR = "#4b94e3";
+const SUCCESS_COLOR = "#33bd51";
+
+/**
+ * Send a warning message to the target channel.
+ */
+export const sendWarn = (response: string, message: Message) => {
```

Comment:
Would be better to rewrite this method as this, me thinks:
```suggestion
export const sendWarn = (channel: TextChannel, response: string) => {
```